bytecomp/lambda.ml

```diff
@@ -263,7 +263,7 @@ and lambda_event =
   { lev_loc: Location.t;
     lev_kind: lambda_event_kind;
     lev_repr: int ref option;
-    lev_env: Env.summary }
+    lev_env: Env.t }
```

Comment:
Is this a wise choice? I thought that `Env.summary` was designed precisely to be the "compact" part of the environment for use in debug information.

Reply:
I'm not sure where you got that from, but you probably have better sources than I do. I'm also not sure what you're worried about: we still "summarize" these environment when transforming lambda to the next AST, they are never marshalled or anything.
Considering we still have modifications to do on the environment at this point, I think it make more sense to actually modify `Env.t`s than the summaries.

Reply:
One wouldn't think of looking there but I got it from [env.mli](https://github.com/ocaml/ocaml/blob/trunk/typing/env.mli#L242-L245) :-)

Reply:
Indeed, I hadn't looked *there*.
Then again, depends how you interpret it. Does "compact" refer to the marshalled representation that end up in `.cmo` files (or similar),  or to something else? My bet is on the former, in which case I believe my change is fine.